budgetApp1/budgetApp1/BudgetViewController.swift

```diff
@@ -9,49 +9,73 @@ import UIKit
 
 class BudgetViewController: UIViewController {
     
-    private var viewModel: BudgetViewModel? {
-        didSet {
-            fillUI()
-        }
-    }
+    // MARK: - UISpecs Dependency
+    private let uiSpecs = UISpecs()
```

Comment:
What is UISpecs? Please give this a more descriptive name

Reply:
Deleted it because I now use the colors in my assets folder